### DISCUSSION TITLE
Job for strongswan.service failed because the control process exited with error code.

### DISCUSSION THREAD

**Discussion by Aramousa** [strongswan/strongswan#399] [General]
Somebody can help me ?
After configure Strongswan & XL2TPD , on **systemctl restart strongswan** command it occurred in problem.
the "journalctl" ccontains :

` [root@localhost system]# journalctl -xe 
May 12 12:32:49 localhost.localdomain charon-systemd[9819]: HA config misses lo>
May 12 12:32:49 localhost.localdomain charon-systemd[9819]: no script for ext-a>
May 12 12:32:49 localhost.localdomain charon-systemd[9819]: loaded plugins: cha>
May 12 12:32:49 localhost.localdomain charon-systemd[9819]: dropped capabilitie>
May 12 12:32:49 localhost.localdomain charon-systemd[9819]: spawning 16 worker >
May 12 12:32:49 localhost.localdomain swanctl[9836]: no files found matching '/>
May 12 12:32:49 localhost.localdomain swanctl[9836]: abort initialization due t>
May 12 12:32:49 localhost.localdomain systemd[1]: strongswan.service: Control p>
May 12 12:32:49 localhost.localdomain charon-systemd[9819]: SIGTERM received, s>
May 12 12:32:49 localhost.localdomain systemd[1]: strongswan.service: Failed wi>
-- Subject: Unit failed
-- Defined-By: systemd
-- Support: https://access.redhat.com/support
--
-- The unit strongswan.service has entered the 'failed' state with result 'exit>
May 12 12:32:49 localhost.localdomain systemd[1]: Failed to start strongSwan IP>
-- Subject: Unit strongswan.service has failed
-- Defined-By: systemd
-- Support: https://access.redhat.com/support
--
-- Unit strongswan.service has failed.
--
-- The result is failed.
`

the error occurred on CentOS 8.

**Comment by tobiasbrunner** [comment]
Please at least post the complete log (that cropped output doesn't help at all). Make sure your config is in the correct files (I think the RPM packages put them in `/etc/strongswan`, not `/etc` directly).
